### DISCUSSION TITLE
how to prepare for UNO

### DISCUSSION THREAD

**Discussion by proficad** [unoplatform/uno#9348] [Q&A]
I ran command 
`uno-check` 
and got error 
```
Fix failed - Workload Install failed: `dotnet workload install --from-rollback-file
"C:\Users\vjedl\AppData\Local\Temp\maui-check-2c6ad788\workload.json" android ios maccatalyst tvos macos wasm-tools
--source "https://api.nuget.org/v3/index.json" --source "https://aka.ms/dotnet6/nuget/index.json"`
```

What can I do?

Thanks
Vaclav

**Comment by francoistanguay** [comment]
Are you running the latest uno-check 1.5.1?
https://twitter.com/UnoPlatform/status/1550231604232077313?s=20&t=3bNB6PfsgM6gmh5_hOYKeg

**Comment by proficad** [comment]
yes

**Reply by francoistanguay** [reply]
Are you running on Windows? Wanting to use latest Uno? If so, you need to run uno-check --pre as explained here:
https://platform.uno/docs/articles/get-started-vs-2022.html?tabs=windows

We'll need a bit more context in order to help.

**Comment by proficad** [comment]
https://www.screencast.com/t/31l7c6bcWJmq

**Comment by proficad** [comment]
I know that page, I had followed instructions on that page.

**Comment by ebariche** [comment]
@proficad Two things :

1- I see you have NET 7 installed, it is going to interfere with the workload installation because `dotnet workload install` runs in the context of NET 7 and you are trying to install NET 6 workloads. Would you be able to uninstall it ?

2- You need to use `uno-check --pre` as you are using VS Preview !

**Comment by proficad** [comment]
I uninstalled "VS2022 preview" and installed "VS2022".
uno-check reports "WSL not available" (I ihad installed WSL before).

Thanks
Vaclav

**Reply by ebariche** [reply]
Could you run `wsl -l` and paste the output here ?

Is everything else ok ?

**Comment by proficad** [comment]
Strange, wsl -l gives list of commands, as if it was wrong syntax:
https://www.screencast.com/t/NPWNLtSB

**Comment by proficad** [comment]
anyway, I was able to build and run the "hello world" app.
Thank you for your help !
Vaclav
